recipes/hmftools-purple/meta.yaml

```diff
@@ -1,17 +1,18 @@
-{% set version = "4.0.2" %}
-{% set sha256 = "fb534734b4a0833c8d68591e009caea5b654714211a95f23570c1fca9bfa57a9" %}
+{% set version = "4.1_beta" %}
+{% set beta_suffix = ".1" %}
+{% set sha256 = "aa5b1ec3c0e67027750fce6ac71bc31d55c37405c8e00571594d58c837e74a51" %}
 
 package:
   name: hmftools-purple
   version: '{{ version }}'
 
 source:
-  url: https://github.com/hartwigmedical/hmftools/releases/download/purple-v{{ version }}/purple_v{{ version }}.jar
+  url: https://github.com/hartwigmedical/hmftools/releases/download/purple-v{{ version }}/purple_v{{ version }}{{ beta_suffix }}.jar
   sha256: '{{ sha256 }}'
 
 build:
   noarch: generic
-  number: 0
+  number: 1
```

Comment:
you don't need to bump if you increase the version number

Reply:
Gotcha, thanks for the tip